docs/source/api/classes/xgi.classes.hypergraph.Hypergraph.rst

```diff
@@ -46,3 +46,4 @@
 
       ~Hypergraph.copy
       ~Hypergraph.dual
+      ~Hypergraph.cleanup
```

Comment:
Is this typo? This shouldn't be here, let's leave it under "methods that modify the structure".

Reply:
Depending on the `in_place` argument, a hypergraph may be returned. Happy to omit, however.

Reply:
Oh snap, you're right. Uhhh let's just omit it for now.

Reply:
Okay!